src/ring_theory/polynomial/hermite.lean

```diff
@@ -124,5 +137,72 @@ begin
 end
 
 end coeff
+end polynomial
+
+/-! ### Lemmas about `polynomial.hermite_gauss` -/
+
+section gaussian
+
+/-- The real Gaussian function -/
+def gaussian : ℝ → ℝ := λ x, real.exp (-(x^2 / 2))
```

Comment:
Keeping this one might be justifiable, especially if there are other existing places in mathlib that use it.

If we did do that it would want to live it it's own file somewhere.

Reply:
Maybe in `analysis/special_functions/gaussian.lean` which is about integrals of gaussians?

Reply:
I'd say focus on the rest of the PR for now. It concerns me a little that file uses a mixture of `real.exp` and `complex.exp`, suggesting a `gaussian` definition may not be so useful after all.

Reply:
I'm leaning towards just taking out `gaussian`.

Reply:
@eric-wieser I think the rest of the PR is mainly held up by this decision now. Thoughts?

Reply:
(decided to remove after offline discussion)